tests/test_entry.py

```diff
@@ -0,0 +1,43 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Entry, PasswordEntry, SearchEntry
+from tests.utils import WindowTestCase
+
+
+class EntryTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.entry = Entry()
+        self.password_entry = PasswordEntry()
+        self.search_entry = SearchEntry()
```

Comment:
True, I do not need these two variables. I just instanced them here at first, but don't need it.